iamclient/iamclient.go

```diff
@@ -68,93 +87,133 @@ func New(config *config.Config, cryptocontext *cryptutils.CryptoContext, insecur
 		}
 	}()
 
+	if err = client.openGRPCConnection(); err != nil {
+		return nil, aoserrors.Wrap(err)
+	}
+
+	if !insecure {
+		if ch, err := client.SubscribeCertChanged(config.CertStorage); err != nil {
+			return nil, aoserrors.Wrap(err)
+		} else {
+			client.tlsCertChan = ch
```

Comment:
Pass cert channel as parameter instead of private member.

Reply:
fixed in aos_common